CONTRIBUTING.md

```diff
@@ -159,18 +159,6 @@ Some scripts require third-party tools, but most of these are installed during t
 
 Most scripts (except `build.sh`) output results in `target/<name>`, e.g. `target/docs`.
 
-### False positive findbugs violation
```

Comment:
Maybe leave this line and add instructions for using dev-support/findbugsExcludeFile.xml file to exclude false positives?

Reply:
I don't think this level of detail belongs to this guide, intended for new contributors.

The example given in the guide was actually a bug, shouldn't have been suppressed.